admin/netdata/files/netdata.init

```diff
@@ -14,7 +14,17 @@ start_service() {
 	mkdir -m 0755 -p /var/log/netdata
 	chown nobody /var/log/netdata
 	procd_open_instance
-	procd_set_param command $APPBINARY -D -c $CONFIGFILE
+
+        local cloud_claim_token cloud_claim_rooms
+        config_get cloud_claim_token "config" "cloud_claim_token" ""
+        config_get cloud_claim_rooms "config" "cloud_claim_rooms" ""
```

Comment:
```suggestion
	local cloud_claim_token cloud_claim_rooms
	config_get cloud_claim_token "config" "cloud_claim_token"
	config_get cloud_claim_rooms "config" "cloud_claim_rooms"
```

STOP MIXING THE USAGE OF SPACES AND TABS
and put them above `mkdir -m 0755 -p /var/cache/netdata`